backend/battles/helpers.py

```diff
@@ -14,3 +14,22 @@ def save_pokemon_in_team(selected_team):
                 defense=data["defense"],
                 hp=data["hp"],
             )
+
+
+def pokemon_attr_exceeds_limit(team):
+    limit = 600
+
+    team_data = [get_pokemon_stats(pokemon) for pokemon in team]
+    pkn_points = []
```

Comment:
The name here could be improved don't you think?

Reply:
I agree.
`team_data` stores 3 pokemon data dict (including name, id, sprite, attack, defense and hp), does it make sense to name it `team_stats` or `pokemon_stats`?
In regards to `pkn_points` which stores the sum for each pokemon stats (attack + defense + hp) I thought of `sum_pkn_stat`.